.github/workflows/test_and_deploy.yml

```diff
@@ -50,7 +50,6 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Install cibuildwheel
-        # MAKE SURE THIS STAYS IN SYNC WITH THE LOWER GHA cibuildwheel
```

Comment:
Why is the comment removed? The `cibuildwheel` version still needs to be kept in sync, right?

Reply:
That was removed by accident, will readd it.